gulpfile.js

```diff
@@ -175,6 +181,40 @@ gulp.task('scripts:dev:feature-flags', () => {
     .then(contents => writeFile(path.resolve(__dirname, 'demo/feature-flags.js'), contents));
 });
 
+const pathsToConvertToESM = new Set([
+  path.resolve(__dirname, 'src/globals/js/feature-flags.js'),
+  path.resolve(__dirname, 'src/globals/js/settings.js'),
+]);
+
+const covertToESMGulpPlugin = () =>
```

Comment:
small nitpick: function expression name

Reply:
@emyarod Thank you for catching this! Fixed.